redaxo/src/addons/install/lib/api/api_package_upload.php

```diff
@@ -30,7 +30,11 @@ public function execute()
         try {
             if ($upload['upload_file']) {
                 $archive = rex_path::addonCache('install', md5($addonkey . time()) . '.zip');
-                $exclude = [];
+                $exclude = [
+                    '.gitattributes',
+                    '.github',
+                    '.gitignore',
```

Comment:
```suggestion
                    '.gitignore',
                    '.git',
                    '.gitkeep',
```
würde die sicherheitshalber auch mit aufnehmen.

Bei `.gitkeep` bin ich mir aber nicht sicher, weil die ja benutzt werden um sicherzustellen, dass Ordner existieren, wenn wir die jetzt wegnehmen, würde vermutlich kein leerer Ordner mehr im Archive existieren oder?

Reply:
.git wird bereits direkt über den rex_finder gefiltert (per default).
bei .gitkeep bin ich mir auch unsicher. vielleicht eher weglassen.

Reply:
sollten hier auch `.idea` oder `.github` o.ä. auftauchen?